applications/luci-app-openvpn/luasrc/view/openvpn/cbi-select-input-add.htm

```diff
@@ -67,6 +67,18 @@
 		{
 			setTimeout(function(){ output.innerHTML=""}, timeout);
 		}
+	}
```

Comment:
Are you sure that the indentation is correct? The one with the bracket looks strange